src/scripts/main.js

```diff
@@ -1,3 +1,14 @@
 'use strict';
 
-// write code here
+const wall = document.querySelector('.wall');
+const spider = document.querySelector('.spider');
+
+const parentWidth = wall.offsetWidth;
+const parentHeight = wall.offsetHeight;
+const childWidth = spider.offsetWidth;
+const childHeight = spider.offsetHeight;
+
+spider.style.cssText = `
+top: ${(parentWidth - childWidth) / 2}px;
+left: ${(parentHeight - childHeight) / 2}px;
```

Comment:
The calculation for the 'left' property is incorrect. It should use 'parentWidth' instead of 'parentHeight'.